src/scss/trumps/_utilities.scss

```diff
@@ -139,7 +139,8 @@
     color: $color-text;
 }
 
-.u-color-link {
+.u-color-link,
+.u-color-secondary {
```

Comment:
Without a pre-existing `.u-color-primary` class, I'm not sure I understand what this is for, or how we should apply it in future; is there a better name for this?

Reply:
`-secondary` refers to directly to a colour variable within fozzie - as does `-link` ($color-link-default and $color-secondary (both use $blue)).

If the classes were not directly linked to a fozzie colour var I would agree :)

Reply:
Okay, that connection isn't obvious from this instance, so I was concerned it wouldn't be obvious for other developers. I'll approve in this instance though.